src/monitor/Monitor.ts

```diff
@@ -734,7 +737,7 @@ export class Monitor {
         : this.monitorChains;
 
     const l2TokenForChain = (chainId: number, symbol: string) => {
-      return TOKEN_SYMBOLS_MAP[symbol]?.addresses[chainId];
+      return toAddressType(TOKEN_SYMBOLS_MAP[symbol]?.addresses[chainId], chainId);
```

Comment:
If `TOKEN_SYMBOLS_MAP[symbol]` is undefined then this will throw.

```suggestion
      const token = TOKEN_SYMBOLS_MAP[symbol];
      return isDefined(token) ? toAddressType(token.addresses[chainId], chainId) : undefined;
```